src/footer/index.md

```diff
@@ -30,13 +28,15 @@ nav_categories:
         url: https://discordapp.com/invite/wYezN7v
       - label: GitHub
         url: https://github.com/keep-network/
-copyright_text: >-
-  <p>A Thesis<sup>*</sup> Build</p><p>© 2020 Keep SEZC. All Rights Reserved.</p>
+copyright_text: <p>A Thesis<sup>*</sup> Build</p><p>© 2020 Keep SEZC. All Rights Reserved.</p>
 legal_links:
   - label: Privacy Policy
     url: /privacy-policy
+    is_external_link: false
```

Comment:
Was this necessary?

Reply:
Ah, I'm sure the CMS just forced it in. I get it.

Reply:
The CMS complained it was missing a required field, yah